guides/mn-serverless-function-aws-lambda-graalvm/mn-serverless-function-aws-lambda-graalvm.adoc

```diff
@@ -86,7 +86,7 @@ However, I like to specify it in the console as well:
 
 `example.micronaut.FunctionRequestHandler`
 
-image::lambda-custom-runtime-bookrequest-handler.png[]
+image::lambda-custom-runtime-functionrequest-handler.png[]
```

Comment:
I think lambda-custom-runtime-bookrequest-handler.png can now be deleted?

Reply:
deleted here https://github.com/micronaut-projects/micronaut-guides/pull/1342/commits/06b72abea05394b2437fa085af613fb10333b2d8